source/transactions/transactions.rst

```diff
@@ -744,24 +753,71 @@ format:
 
     { ok : 1, writeConcernError: {code: <Number>, errmsg : "..."} }
 
-Mongos Pinning for Sharded Transactions
----------------------------------------
+Sharded Transactions
+--------------------
+
+MongoDB 4.2 (maxWireVersion 8) introduces support for sharded transactions.
+Sharded transactions support all of the same features as single replica set
+transaction but introduce two new driver concepts: mongos pinning and the
+``recoveryToken`` field.
 
-The server supports sharded transactions starting in MongoDB 4.2 (
-maxWireVersion 8). The server requires that drivers MUST send all commands for
-a single transaction to the same mongos.
+Mongos Pinning
+~~~~~~~~~~~~~~
+
+Drivers MUST send all commands for a single transaction to the same mongos
+(excluding retries of commitTransaction and abortTransaction).
 
 After the driver selects a mongos for the first command within a transaction,
 the driver MUST pin the ClientSession to the selected mongos. Drivers MUST
-send all commands that are part of the same transaction, including
-commitTransaction and abortTransaction and any retries thereof, to the
-same mongos.
+send all subsequent commands that are part of the same transaction (excluding
+retries of commitTransaction and abortTransaction) to the same mongos.
+
+When to unpin
+^^^^^^^^^^^^^
+
+Drivers MUST unpin a ClientSession when a command within a transaction
+fails with a TransientTransactionError (including a server selection error).
+Transient errors indicate that the transaction in question has already been
+aborted or that the pinned mongos is down/unavailable. Unpinning the session
+ensures that a subsequent abortTransaction does not block waiting on a server
+that is unreachable.
+
+Additionally, Drivers MUST unpin a ClientSession after running any
+commitTransaction or abortTransaction attempt regardless if that attempt
+succeeded or failed. After the initial commit or abort attempt, any mongos can
+satisfy a subsequent retry. Note, when the initial attempt fails with a
+retryable writes error, the automatic retry attempt MUST perform normal
```

Comment:
This should simply refer to "retryable error". I believe that term is already used elsewhere and refers back to the original definition in the retryable writes spec.

Reply:
Yep that's right. Fixed.